packages/mdc-button/_variables.scss

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+$mdc-button-baseline-color-on-light: #6200ee;
```

Comment:
This would kind of break a bunch of back-compat right now what with there being separate classes for primary and accent... I know we're planning to get rid of those, but can we do that separately?  (I realize adding variables just to remove them is sort of a bummer though.)